sponsorship/src/test/java/com/kyriba/conference/sponsorship/api/ContractBaseTest.java

```diff
@@ -0,0 +1,30 @@
+package com.kyriba.conference.sponsorship.api;
+
+import io.restassured.module.mockmvc.RestAssuredMockMvc;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.web.context.WebApplicationContext;
+
+
+/**
+ * @author Aliaksandr Samal
+ */
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+@ActiveProfiles("test")
+public class ContractBaseTest
```

Comment:
It seems you don't use this class for inheritance

Reply:
Hm... This class is used to create an auto-generated test which is placed in 'build' folder at API producer side and out of Git control. Should that test be moved and added to source control?

Reply:
Sorry, my bad. I forgot about generated classes in contact testing.